src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -305,9 +305,19 @@ private void visitRequireCall(NodeTraversal t, Node require, Node parent) {
       }
 
       String moduleName = ES6ModuleLoader.toModuleName(loadAddress);
-      Node moduleRef = IR.name(moduleName).srcref(require);
-      parent.replaceChild(require, moduleRef);
       Node script = getCurrentScriptNode(parent);
+
+      // When require("name") is used as a standalone statement (the result isn't used)
+      // it indicates that a module is being loaded for the side effects it produces.
+      // In this case the require statement should just be removed as the goog.require
+      // call inserted will import the module.
+      if (!NodeUtil.isExpressionResultUsed(require) && parent.isExprResult() &&
```

Comment:
"&&" should be on a new line according to https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break